client/src/components/postcall/PostCallOneQuestionPoll.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { OneQuestionPollOptions } from '../../models/ConfigModel';
+import { PartialTheme, PrimaryButton, Stack, Text, Theme } from '@fluentui/react';
+import OneQuestionPollInput from './OneQuestionPollInput';
+import { pollPromptStyle, pollTitleStyle, surveySubmitButtonStyles } from '../../styles/Survey.styles';
+import { useState } from 'react';
+import { submitSurveyResponse } from '../../utils/PostCallUtil';
+
+export interface PostCallOneQuestionPollProps {
+  oneQuestionPollInfo: OneQuestionPollOptions;
+  theme?: PartialTheme | Theme;
+  callId?: string;
+  acsUserId: string;
+}
+
+export const PostCallOneQuestionPoll: React.FunctionComponent<PostCallOneQuestionPollProps> = (
+  props: PostCallOneQuestionPollProps
+) => {
+  const [pollResponse, setPollResponse] = useState<boolean | string | number>();
+
+  return (
+    <Stack horizontalAlign="center" verticalAlign="center">
+      <Text styles={pollTitleStyle}>{props.oneQuestionPollInfo.title}</Text>
+      <Text styles={pollPromptStyle}>{props.oneQuestionPollInfo.prompt}</Text>
+      <OneQuestionPollInput
+        pollType={props.oneQuestionPollInfo.pollType}
+        textInputPlaceholder={props.oneQuestionPollInfo.answerPlaceholder}
+        setPollResponse={setPollResponse}
+      />
+      <PrimaryButton
+        styles={surveySubmitButtonStyles}
+        onClick={() =>
+          submitSurveyResponse(
+            props.acsUserId,
+            pollResponse !== undefined ? pollResponse : undefined,
+            props.callId ? props.callId : undefined
```

Comment:
Are the null checks necessary here? I believe this is logically equivalent to:

```
submitSurveyResponse(props.acsUserId, pollResponse, props.callId);
```

Reply:
Yes they are necessary else the assignment into submitSurveyResponse fails/throws errors

Reply:
`assignment into submitSurveyResponse` Can you clarify, do you mean the API call fails?

Reply:
There's no compiler errors so I don't see a need for the null checks. If there are errors downstream then that business logic must be added there.

![2022-10-12 16_18_29-PostCallOneQuestionPoll tsx - postcall-v2 - Visual Studio Code](https://user-images.githubusercontent.com/43504546/195465172-5b52a60e-7e4b-4571-84f5-726c215989c3.png)

Reply:
Agghhh... not sure what happened, was getting some errors before which I dont remember after the dozen other changes so far.. but this works fine, have made this change!